omniflixhub/assetlist.json

```diff
@@ -39,6 +39,37 @@
       },
       "type_asset": "sdk.coin"
     },
+    {
+      "description": "GATA Yield DAO Governance Token powered by OmniFlixHub",
+      "denom_units": [
+        {
+          "denom": "factory/omniflix1fwphj5p6qd8gtkehkzfgac38eur4uqzgz97uwvf6hsc0vjl004gqfj0xnv/ygata",
+          "exponent": 0
+        },
+        {
+          "denom": "ygata",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/omniflix1fwphj5p6qd8gtkehkzfgac38eur4uqzgz97uwvf6hsc0vjl004gqfj0xnv/ygata",
+      "name": "Yield GATA",
+      "display": "yGATA",
+      "symbol": "yGATA",
```

Comment:
I notice this contains lowercase letters, meaning this is derived from another asset. You could please define the trace to GATA?

Reply:
Sorry for the inconvenience. The denom has been corrected, and there’s no other GATA. Thank you.